### PR TITLE
refactor: type multi language controls

### PR DESCRIPTION
**Pull request by derschnee68**
resolves DEV-3343

### CODE REVIEW

File: libs/vre/shared/app-string-literal/src/index.ts
Comment (irmastnt) [inline libs/vre/shared/app-string-literal/src/index.ts:1]: Move multi language form components out from string literal

File: libs/vre/shared/app-string-literal/src/lib/multi-language-form.service.ts [hunk 1]
```diff
@@ -1,43 +1,42 @@
...
-      this.formArray.controls.find(
-        control => control.value.language === this.availableLanguages[this.selectedLanguageIndex]
-      ) as FormControl
-    ) // TODO as FormControl = bad pattern, should be refactored, also form should be with Typed values
-      .get('value') as FormControl;
+  get selectedLanguageControl() {
+    return this.selectedFormControl.controls.value;
   }
 
   get invalidErrors() {
```
Comment (irmastnt) [inline libs/vre/shared/app-string-literal/src/lib/multi-language-form.service.ts:32]: check this getter usages. undefined is not expected in some places

**Review by irmastnt — CHANGES_REQUESTED** [review]

File: libs/vre/shared/app-string-literal/src/lib/multi-language-form.service.ts (hunk 1 above)
Comment (derschnee68) [reply libs/vre/shared/app-string-literal/src/lib/multi-language-form.service.ts:32]: it is intended to be undefined, but the check is done before

File: libs/vre/shared/app-string-literal/src/index.ts
Comment (derschnee68) [reply libs/vre/shared/app-string-literal/src/index.ts:1]: it is actually the string-literal component that will be removed and folder will be renamed as soon as the last refactor is done (next PR)

**Pull request merged**